sui/utils/sign-utils.js

```diff
@@ -40,6 +41,13 @@ function getWallet(chain, options) {
         }
     }
 
+    const client = getSuiClient(chain, options.rpc);
+
+    if (options.privateKey === 'ledger') {
```

Comment:
Make use of the `privateKeyType` instead, that makes more sense instead of reusing the `privateKey`. We should change EVM scripts to also have include a privateKeyType (since we could have hex, mnemonic, or ledger types) as a better design